IDEAS/Buildings/Components/OuterWall.mo

```diff
@@ -88,7 +104,9 @@ protected
     redeclare package Medium = Medium,
     final table=coeffsCp,
     final azi=aziInt,
-    Cs=Cs,
+    Cs=if not Use_custom_Cs and sim.interZonalAirFlowType == IDEAS.BoundaryConditions.Types.InterZonalAirFlow.TwoPorts
```

Comment:
Perhaps make a protected variable `Cs_int` with documentation string that explains what the rationale is behind this implementation.